packages/react-select/.babelrc.json

```
@@ -0,0 +1,3 @@
+{
+  "plugins": ["annotate-pure-calls", "@babel/transform-react-pure-annotations"]
```

```suggestion
  "presets": ["@griffel"],
  "plugins": ["annotate-pure-calls", "@babel/transform-react-pure-annotations"]
```

This should use Griffel preset :)